everyvoice/preprocessor/preprocessor.py

```diff
@@ -1042,6 +1056,19 @@ def preprocess(  # noqa: C901
                     with open(self.save_dir / "summary.txt", "w", encoding="utf8") as f:
                         f.write(report)
                         f.write("\n")
+
+                    # Save missing files list to a separate file
+                    if self.missing_files_list:
+                        with open(
+                            self.save_dir / "missing_files.txt", "w", encoding="utf8"
```

Comment:
I think maybe this should be time stamped or somehow named according to the run, since this will just overwrite the file otherwise.